server/services/gameCreate.js

```diff
@@ -141,6 +143,11 @@ module.exports = class GameCreateService {
             this.conversationService.createConversationAllPlayers(game);
         }
 
+        if (isCustomGalaxy) {
+            game.settings.galaxy.starsPerPlayer = game.galaxy.stars.length / game.settings.general.playerLimit;
```

Comment:
How will this handle scenarios where the resulting `starsPerPlayer` is a decimal?

For example, a 30 star galaxy with 8 players yields a `starsPerPlayer` value of `3.75`.

Reply:
I guess floor the number and then it auto invalidates the galaxy